cypress/commons/constants/generic/IdConstants.js

```diff
@@ -159,11 +159,16 @@ export const GENERIC_SELECTORS = {
         fileTypeSelectContainer: '[data-cy=table-export-file-type-container]',
         fileTypeSelectOptionByExtension: '[data-cy=table-export-file-type-select-option-$EXTENSION]',
       },
+      toolbar: {
+        fullscreenButton: '[data-cy=grid-fullscreen-button]',
+      },
       header: '[class=ag-header-container]',
       colByName: '[col-id=$COLNAME]',
       rowsContainer: '[class=ag-center-cols-container]',
       row: '.ag-row',
       rowByIndex: '[row-index="$ROWINDEX"]',
+      fullscreenTable: '[data-cy=fullscreen-table]',
+      scroll: '[class=ag-body-vertical-scroll-viewport]',
```

Comment:
if I'm not mistaken, we don't need this constant anymore

Reply:
DONE